schema.graphql

```diff
@@ -1,369 +1,319 @@
-type Announcement {
-  announcementId: String!
-  company: String!
-  id: ID!
-  locationId: Int!
-  message: String!
-  people: String!
-  publishOn: UTCDateTime!
+# This file was generated based on "schema.json". Do not edit manually.
```

Comment:
This is a big change, was the old version of this generated from the rails endpoint? Might be nice to build this once before changing the schema and commit that on its own. Just so we can have a cleaner diff on this `feat: add graphql high score query` commit

Reply:
Yup good idea, pulled out the initial regeneration into a8cfdba